app/assets/javascripts/modules/calendars/allocations.es6

```diff
@@ -16,14 +16,18 @@
             text: 'Fullscreen',
             click: this.fullscreenClick.bind(this)
           }
-        }
+        },
+        selectable: true
       };
 
       this.eventChanges = [];
       this.isFullscreen = false;
       this.actionPanel = $('[data-action-panel]');
       this.$savedChanges = $('.js-saved-changes');
       this.$form = $('.js-changes-form');
+      this.$holidayForm = $('.js-holiday-form');
```

Comment:
Yeah I'm all for that jquery stuff. I hated it in the beginning thinking it was like prefixing the type (`strName`) but it's really really good for readibility!